### ISSUE TITLE
Getting Error: Cannot find module '@babel/runtime/core-js/object/create'

### ISSUE THREAD

**Issue by Michaeltym** [slevithan/xregexp#262]
I'm using version `4.2.2` and getting error `Error: Cannot find module '@babel/runtime/core-js/object/create'`

Here is the code (From README)
```
const XRegExp = require('xregexp');

const date = XRegExp(
  `(?<year>  [0-9]{4} ) -?  # year
   (?<month> [0-9]{2} ) -?  # month
   (?<day>   [0-9]{2} )     # day`, 'x');

let match = XRegExp.exec('2017-02-22', date);

console.log(match.year);
```

Can confirm that `4.2.0` is working

**Comment by cronvel** [comment]
Same problem here.

**Comment by lorenc-tomasz** [comment]
@josephfrazier The https://github.com/josephfrazier/xregexp/commit/30338174bfa5b555e200e6f9ad15f1f7af2388ba is very ugly ;)

**Comment by josephfrazier** [comment]
@lorenc-tomasz 	it sure is ugly, but I think it will work for now ;) Ideally we'd upgrade to a non-beta version (looks like 7.2.0 is out now) but that might take longer

**Comment by slevithan** [comment]
Thanks for the reports. And thanks for the quick fix, @josephfrazier. Should now be fixed in v4.2.3. Broken version v4.2.2 has been deprecated on npm.

**Comment by josephfrazier** [comment]
Sure thing! I'll probably try to upgrade all our Babel stuff to non-beta sometime this week
